src/avatars/avatar-stack.scss

```diff
@@ -56,7 +56,7 @@
     // stylelint-enable selector-max-type
 
     // Account for 4+ avatars
-    &:nth-child(n+4) {
+    &:nth-child(n + 4) {
```

Comment:
Weird it did this, I don't care either way if the linter doesn't complain and sass still compiles it correctly.

Reply:
Yeh it seems strange but shouldn't cause any issues.